src/plugins/intel_cpu/src/emitters/snippets/aarch64/kernel_executors/gemm_copy_b.cpp

```diff
@@ -154,9 +163,9 @@ void GemmCopyBKaiKernelExecutor::execute(const GemmCopyBKaiKernelExecutor* execu
                                                          K,
                                                          nr,
                                                          kr,
-                                                         sr,                           // Packing arguments
-                                                         N * sizeof(float),            // RHS stride
-                                                         src_ptr,                      // RHS
+                                                         sr,                      // Packing arguments
+                                                         stride * sizeof(float),  // RHS stride accounting for layout
```

Comment:
You can store `stride` in config with already applied data size: we can pass `stride * sizeof(float)` to `config.update(..)` in `update_config`

Reply:
Done